nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/main/java/org/apache/nifi/processors/mongodb/GetMongo.java

```diff
@@ -85,6 +92,22 @@
             .allowableValues(YES_PP, NO_PP)
             .addValidator(Validator.VALID)
             .build();
+    static final PropertyDescriptor USER_NAME = new PropertyDescriptor.Builder()
+            .name("User Name")
+            .displayName("username")
+            .description("User for mongodb authentication")
+            .required(false)
+            .addValidator(Validator.VALID)
+            .sensitive(false)
+            .build();
+
+    static final PropertyDescriptor PASSWORD = new PropertyDescriptor.Builder()
+            .name("Password")
```

Comment:
The `name` value should match the formatting of the _username_ field and a `displayName` should also be present.